internal/setup/setup.go

```diff
@@ -151,14 +173,16 @@ func (r *Release) validate() error {
 			for newPath, newInfo := range new.Contents {
 				if old, ok := paths[newPath]; ok {
 					oldInfo := old.Contents[newPath]
+					// Note that if extracting content (CopyPath or GlobPath)
+					// from the same package the content can never be in conflict.
 					if !newInfo.SameContent(&oldInfo) || (newInfo.Kind == CopyPath || newInfo.Kind == GlobPath) && new.Package != old.Package {
 						if old.Package > new.Package || old.Package == new.Package && old.Name > new.Name {
 							old, new = new, old
 						}
 						return fmt.Errorf("slices %s and %s conflict on %s", old, new, newPath)
 					}
 				} else {
-					if newInfo.Kind == GlobPath {
+					if newInfo.Kind == GlobPath || newInfo.Kind == GeneratePath {
```

Comment:
Related to earlier comment.

Reply:
Let's discuss it in https://github.com/canonical/chisel/pull/143#discussion_r1671911925.